src/components/homepageGallery/GalleryCard/GalleryCard.module.css

```diff
@@ -12,9 +24,15 @@
   text-decoration: none;
   transition: linear 0.2s;
 }
+
 .gallery_card:hover {
   box-shadow: 0 0.7rem 1rem rgba(0, 0, 0, 0.9);
 }
 .gallery_card img {
   width: 100%;
 }
+
+.gallery_card img:hover {
+  width: 100%;
+  animation: bounce 0.4s ease infinite;
+}
```

Comment:
This functions for the most part, but it is odd having to hover the image within the tile because it causes two separate hover states. Instead of triggering the animation on `img:hover`, you can trigger it during the `.gallery_card:hover`.

Additionally, reassigning `width: 100%` on the image is unnecessary. :)

```
.gallery_card:hover {
  box-shadow: 0 0.7rem 1rem rgba(0, 0, 0, 0.9);
}

.gallery_card img {
  width: 100%;
}

.gallery_card:hover img {
  animation: bounce 0.4s ease infinite;
}
```